dataset_interfaces/interface.py

```diff
@@ -338,23 +338,25 @@ def _filler_size(is_q: bool, is_p2q: bool):
         filler[-1] = 0
         return filler
 
-    def tokens_to_answer(self, context: List, example: TestExample, timestamps: List):
+    def tokens_to_answer(self, test_context: List[Dict[str, str]], full_context: List[Dict[str, str]], example: TestExample):
         encoding = tiktoken.get_encoding("cl100k_base")
-
         num_tokens = num_characters = 0
+
         # Get most relevant line, and any characters after that statement.
         script_answer_index, answer_end_char = self.answer_statement_idx(example)
-        target_timestamp = timestamps[script_answer_index].__str__()
         relevant_line = example.script[script_answer_index]
 
+        timestamp_idx = search_context(test_context, relevant_line)
+        target_timestamp = test_context[timestamp_idx]["timestamp"].__str__()
```

Comment:
Type `List[Dict[str, str]]` made me think of the standard OpenAI Context, but this is more like `List[dict]`, or `List[Dict[str, Any]]` if you want to be more precise.